docs/CONTRIBUTING.md

```diff
@@ -1,25 +1,46 @@
 * Create a separate "issue" for each bug you are reporting and each feature you are requesting.
-* Do not use the issues page for things other than bug reports and feature requests. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead.
+* Choose the correct issue template and fill it out to the best of your ability.
+* Do not use the issues page for things other than what match the provided issue templates. Use the [discussions page](https://github.com/endless-sky/endless-sky/discussions) instead. Examples of what the discussion pages would be used for include topics that are not looking for a resolution (as the point of an issue is to have it be resolved) or topics that are extremely broad and not befitting of an issue because there is no single resolution to the topic.
+
+If requesting a new feature, first ask yourself: will this make the game more fun or interesting within the existing scope of the game? Remember that this is not a simulator game; while you may find such games fun or interesting, that is not the type of game that Endless Sky is trying to be.
+
+### Closing issues
+
+If you believe your issue has been resolved, you can close the issue yourself. If your issue gets closed because a PR was merged, and you are not satisfied, please do not reopen your issue. Please open a new issue and explain the gap that you feel still needs to be addressed.
 
-If requesting a new feature, first ask yourself: will this make the game more fun or interesting? Remember that this is a game, not a simulator. Changes will not be made purely for the sake of realism, especially if they introduce needless complexity or aggravation.
+## Pull requests
 
-## Posting pull requests
+### Posting pull requests
 
 If you are posting a pull request, please:
 
+* Find if your pull request solves an existing issue. Note that just because an issue is open does not guarantee that it is something we will accept, as the number of issues we receive means that not all of them have been thoroughly discussed, and there may be open issues for features that we decide we do not want.
 * Do not combine multiple unrelated changes.
 * Check the diff and make sure the pull request does not contain unintended changes.
 * If changing the C++ code, follow the [coding standard](https://endless-sky.github.io/styleguide/styleguide.xml).
+* Fill out the pull request template to the best of your ability. GitHub's pull request template features are not as thorough as issue templates. You are able to delete the entire pull request template and put in whatever you want; please do not do this. The purpose of the pull request template is to help facilitate the review, approval, and merging of pull requests.
 
 If proposing a major pull request, start by posting an issue and discussing the best way to implement it. Often the first strategy that occurs to you will not be the cleanest or most effective way to implement a new feature.
```

Comment:
can we change this from major to a more descriptive term, as small features might want to have an issue assigned to it too